diff --git a/Users/martin/git/rxjs/spec/helpers/marble-testing.ts b/test/helpers/marble-testing.ts
index 8b3c6a9..fffeaf2 100644
--- a/Users/martin/git/rxjs/spec/helpers/marble-testing.ts
+++ b/test/helpers/marble-testing.ts
@@ -1,9 +1,8 @@
-///<reference path='../../typings/index.d.ts'/>
-import {Observable} from '../../dist/cjs/Observable';
-import {SubscriptionLog} from '../../dist/cjs/testing/SubscriptionLog';
-import {ColdObservable} from '../../dist/cjs/testing/ColdObservable';
-import {HotObservable} from '../../dist/cjs/testing/HotObservable';
-import {observableToBeFn, subscriptionLogsToBeFn} from '../../dist/cjs/testing/TestScheduler';
+import {Observable} from 'rxjs/Rx';
+import {SubscriptionLog} from 'rxjs/testing/SubscriptionLog';
+import {ColdObservable} from 'rxjs/testing/ColdObservable';
+import {HotObservable} from 'rxjs/testing/HotObservable';
+import {observableToBeFn, subscriptionLogsToBeFn} from 'rxjs/testing/TestScheduler';
 
 declare const global: any;
 
@@ -41,4 +40,4 @@ export function time(marbles: string): number {
     throw 'tried to use time() in async test';
   }
   return global.rxTestScheduler.createTime.apply(global.rxTestScheduler, arguments);
-}
\ No newline at end of file
+}
